aliases/README.md

```diff
@@ -76,4 +76,42 @@ OMB_ALIAS_PACKAGE_MANAGER_SUDO=
 | `tinit`  | `terraform init`             |
 | `tplan`  | `terraform plan`             |
 | `tapply` | `terraform apply`            |
-| `tfmt`   | `terraform fmt`              | 
+| `tfmt`   | `terraform fmt`              |
+
+## alias:Docker
```

Comment:
Can you put this section before `alias:package-manager`? Alphabetical order is useful for finding the corresponding section.

Reply:
Thanks, I appreciate sorting too. I also added backticks (\<code>alias_name\</code>) around the alias names too. ohmyzsh used a different format.